### PR TITLE
Delegate tracing install to eventing-kafka

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

as per title

/hold

/cc @lberk

### CODE REVIEW

File: test/eventing-kafka.bash [hunk 1]
```diff
@@ -22,12 +22,12 @@ function upstream_knative_eventing_kafka_e2e {
...
 
-  logger.info 'Installing Tracing'
-  install_tracing
-
   # shellcheck disable=SC1090
   source "${KNATIVE_EVENTING_KAFKA_HOME}/openshift/e2e-common.sh"
 
+  logger.info 'Installing Tracing'
+  install_tracing
+
```
Comment (aliok) [inline test/eventing-kafka.bash:30]: +1 on this

File: test/lib.bash [hunk 2]
```diff
@@ -266,7 +265,6 @@ function teardown {
   fi
   logger.warn "Teardown 💀"
   teardown_serverless
-  teardown_tracing
```
Comment (aliok) [inline test/lib.bash:269]: Why do you delete this one?

Comment (matzew) [reply test/lib.bash:269]: ah, I wanna add that to the upstream too ....  PR is still on hold :-)

**Comment by mgencur** [comment]
Hey @matzew , this should probably fix this issue from the nightly run on serverless-operator tests? https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-knative-serverless-operator-main-4.7-e2e-aws-ocp-47-continuous/1371793566438789120#1:build-log.txt%3A2477

File: hack/lib/strimzi.bash [hunk 3]
```diff
@@ -85,7 +84,7 @@ spec:
 EOF
```
Comment (matzew) [inline hack/lib/strimzi.bash:85]: @slinkydeveloper here, I am doing the creates .... /cc @scholzj

File: hack/lib/strimzi.bash [hunk 4]
```diff
@@ -4,18 +4,17 @@ function install_strimzi_operator {
   strimzi_version=$(curl https://github.com/strimzi/strimzi-kafka-operator/releases/latest |  awk -F 'tag/' '{print $2}' | awk -F '"' '{print $1}' 2>/dev/null)
   header "Installing Strimzi Kafka operator"
   oc create namespace kafka
-  oc -n kafka apply --selector strimzi.io/crd-install=true -f "https://github.com/strimzi/strimzi-kafka-operator/releases/download/${strimzi_version}/strimzi-cluster-operator-${strimzi_version}.yaml"
   curl -L "https://github.com/strimzi/strimzi-kafka-operator/releases/download/${strimzi_version}/strimzi-cluster-operator-${strimzi_version}.yaml" \
   | sed 's/namespace: .*/namespace: kafka/' \
-  | oc -n kafka apply -f -
+  | oc -n kafka create -f -
```
Comment (cardil) [inline hack/lib/strimzi.bash:9]: Why replacing `apply` with `create`? Those scripts should be safe to run multiple times. Create will fail for second run.

**Review by cardil — CHANGES_REQUESTED** [review]

File: hack/lib/strimzi.bash (hunk 4 above)
Comment (scholzj) [reply hack/lib/strimzi.bash:9]: `apply` cannot handle big CRDs.

**Pull request closed without merging**